packages/paste-core/components/form/stories/index.stories.tsx

```diff
@@ -0,0 +1,425 @@
+};
+
+export const FixedWidthForm: StoryFn = () => {
+  const seed = useUIDSeed();
+
+  return (
+    <Form maxWidth="size30" aria-labelledby={seed('form-heading')}>
+      <Heading id={seed('form-heading')} as="h2" variant="heading20" marginBottom="space0">
+        {/* Added UID to the heading because forms are HTML Landmarks. Axe was failing because in the stacked view, there were 4 forms with the same name and landmarks have to have unique names. */}
+        Profile settings {useUID()}
+      </Heading>
+      <Paragraph marginBottom="space0">Please add information to your Twilio profile.</Paragraph>
+      <FormSection>
+        <FormSectionHeading>Contact information</FormSectionHeading>
+        <FormSectionDescription>Please add the details of your contact information.</FormSectionDescription>
+        <FormControl>
+          <Label htmlFor={seed('phone-number')}>Phone number</Label>
+          <Input id={seed('phone-number')} type="tel" insertAfter={<InformationIcon decorative />} />
+        </FormControl>
+        <FormControl>
+          <Label htmlFor={seed('email')}>Email address</Label>
+          <Input id={seed('email')} type="email" />
+        </FormControl>
+      </FormSection>
+      <FormSection>
+        <FormSectionHeading>Shipping address</FormSectionHeading>
+        <FormControlTwoColumn>
+          <FormControl>
+            <Label htmlFor={seed('first-name')}>First name</Label>
+            <Input id={seed('first-name')} type="text" />
+          </FormControl>
+          <FormControl>
+            <Label htmlFor={seed('last-name')}>Last name</Label>
+            <Input id={seed('last-name')} type="text" />
+          </FormControl>
+        </FormControlTwoColumn>
+        <FormControl>
+          <Label htmlFor={seed('address')}>Address</Label>
+          <Input id={seed('address')} type="text" />
+        </FormControl>
+        <FormControl>
+          <Label htmlFor={seed('address-2')}>Apartment, suite, etc. (Optional)</Label>
+          <Input id={seed('address-2')} type="text" />
+        </FormControl>
+        <FormControl>
+          <Label htmlFor={seed('city')}>City</Label>
+          <Input id={seed('city')} type="text" />
+        </FormControl>
+        <FormControl>
+          <Combobox labelText="Country" placeholder="Please select a country" items={countriesList} />
+        </FormControl>
+        <FormControlTwoColumn>
+          <FormControl>
+            <Label htmlFor={seed('state')}>State</Label>
+            <Select id={seed('state')}>
+              {statesList.map(({value, name}) => (
+                <Option value={value} key={value}>
+                  {name}
+                </Option>
+              ))}
+            </Select>
+          </FormControl>
+          <FormControl>
+            <Label htmlFor={seed('zip')}>Zip code</Label>
+            <Input id={seed('zip')} type="text" />
+          </FormControl>
+        </FormControlTwoColumn>
+      </FormSection>
+      <FormControl>
```

Comment:
thought: I notice that having a dangling `FormControl` after a `FormSection` makes that control seem still related to the previous section spacing-wise, since the two are still the standard space80 apart (the section ends before **Select payment method.** 
<img width="348" alt="image" src="https://user-images.githubusercontent.com/77117846/211395071-c409f487-e5ed-4616-91a6-91fc5a71466a.png">

Is that something we should guide people not to do in the docs?

Reply:
@kelseycavitt What do you think about this? Should we add guidance about this in the docs?